watertap/unit_models/translators/translator_asm1_adm1.py

```diff
@@ -184,9 +184,12 @@ def CODd(blk, t):
             doc="COD demand minus readily biodegradable substrate",
         )
         def CODd_step1(blk, t):
-            return (
-                blk.inter_S_S[t]
-                == blk.properties_in[t].conc_mass_comp["S_S"] - blk.CODd[t]
+            return blk.inter_S_S[t] == 0.5 * (
+                (blk.properties_in[t].conc_mass_comp["S_S"] - blk.CODd[t])
+                + sqrt(
+                    (blk.properties_in[t].conc_mass_comp["S_S"] - blk.CODd[t]) ** 2
+                    + eps
+                )
             )
```

Comment:
We should use the IDAES soft absolute value if that's what's really needed here.

Reply:
Its the max(value, 0) formulation if the value is negative I needed it to be 0